### ISSUE TITLE
fwupdate needs efivar master to build

### ISSUE THREAD

**Issue by superm1** [rhboot/efivar#96]
Since fwupdate 10 was released today, I noticed that efivar 32 won't build against it.

```
libfwup.c: In function 'make_ux_capsule_entry':
libfwup.c:733:30: error: 'efi_guid_ux_capsule' undeclared (first use in this function); did you mean 'efi_guid_ux_capsule_guid'?                                                                               
  fwup_ux_capsule.esre.guid = efi_guid_ux_capsule;
                              ^~~~~~~~~~~~~~~~~~~                                                                                                                                                              
                              efi_guid_ux_capsule_guid
libfwup.c:733:30: note: each undeclared identifier is reported only once for each function it appears in                                                                                                       
libfwup.c: In function 'write_ux_capsule_header':
libfwup.c:1648:11: error: 'efi_guid_ux_capsule' undeclared (first use in this function); did you mean 'efi_guid_ux_capsule_guid'?                                                                              
   .guid = efi_guid_ux_capsule,
           ^~~~~~~~~~~~~~~~~~~                                                                                                                                                                                 
           efi_guid_ux_capsule_guid
libfwup.c: In function 'fwup_set_up_update':                                                                                                                                                                   
libfwup.c:1795:37: error: 'efi_guid_ux_capsule' undeclared (first use in this function); did you mean 'efi_guid_ux_capsule_guid'?
  if (!efi_guid_cmp(&re->esre.guid, &efi_guid_ux_capsule)) {                                                                                                                                                   
                                     ^~~~~~~~~~~~~~~~~~~
                                     efi_guid_ux_capsule_guid                                                                                                                                                  
libfwup.c: In function 'fwup_set_up_update_with_buf':
libfwup.c:1899:37: error: 'efi_guid_ux_capsule' undeclared (first use in this function); did you mean 'efi_guid_ux_capsule_guid'?                                                                              
  if (!efi_guid_cmp(&re->esre.guid, &efi_guid_ux_capsule)) {
                                     ^~~~~~~~~~~~~~~~~~~                                                                                                                                                       
                                     efi_guid_ux_capsule_guid
/build/fwupdate-10/linux/Makefile:90: recipe for target 'libfwup.o' failed                                                                                                                                     
```

This is already fixed in master (https://github.com/rhboot/efivar/commit/cd732494ba7685feaf71b9ee58619ca6aef39fc9) but there hasn't been a new tagged release since September.
Would you mind issuing a new release?

**Comment by daenney** [comment]
Ping @vathpela. It seems 33 has been "released" in 0327a3f84412f55761f7eb80606fb3cfa23f4448 but no tag exists. Could this be fixed? It would be great to get a new efivars into Arch so we can get a newer fwupdate and people with systemd-bootd can start installing firmware upgrades again.

**Comment by superm1** [comment]
34 was tagged today, closing this
